packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte

```diff
@@ -29,15 +68,18 @@
                 <SearchInput bind:value={$nftSearchTerm} />
             </div>
         </div>
-
-        {#if $queriedNfts.length}
-            <div
-                class="grid overflow-scroll sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-5 auto-rows-auto gap-3 2xl:gap-4 pb-1 pr-3 -mr-5"
-            >
-                {#each $queriedNfts as nft}
-                    <NftGalleryItem {nft} />
-                {/each}
-            </div>
+        {#if nftsRows.length}
+            <VirtualList items={nftsRows} let:item>
+                <div
+                    class="grid overflow-scroll sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-5 auto-rows-auto gap-3 2xl:gap-4 pb-1 pr-3 -mr-5"
```

Comment:
doesnt the custom cols and rows interact with all this tailwind breakpointss?

Reply:
What do you mean by interact? You mean conflict? If that's the case, no, they don't conflict